roles/matrix-dynamic-dns/defaults/main.yml

```diff
@@ -0,0 +1,40 @@
+# Whether dynamic dns is enabled
+matrix_dynamic_dns_enabled: false
+
+# The dynamic dns daemon interval
+matrix_dynamic_dns_daemon_interval: '300'
+
+# The docker container to use when in mode
+matrix_dynamic_dns_docker_image: 'linuxserver/ddclient'
```

Comment:
Maybe this should have an explicit `:latest` tag here? This would also allow `matrix_dynamic_dns_docker_image_force_pull` to work as expected.

We also prefer sticking to fixed versions which we know to work, where available. So we may go for something like `v3.9.1-ls44` instead.

Looks like all of these tags supported multiple architectures. This would work nicely for us, without further complications (no need to enable self-building for non-`amd64` or something).

Reply:
Great tips! Done.